pkg/types/event.go

```diff
@@ -73,6 +74,21 @@ func (e *Event) GetMeta(key string) string {
 	return ""
 }
 
+func (e *Event) ParseIPSources() []net.IP {
```

Comment:
do you need to export this, and the other methods on Node?

Reply:
Yes as it used as a helper function.. unless it can be called privately?